Products/ZenModel/migrate/czadminrole.py

```diff
@@ -68,8 +77,12 @@ def cutover(self, dmd):
             roles = [entry['name']
                      for entry in zport.rolesOfPermission(perm)
                      if entry['selected']]
-            if ZEN_MANAGER_ROLE in roles:
-                zport.manage_permission(perm, roles + [CZ_ADMIN_ROLE], 1)
+            if ZEN_MANAGER_ROLE in roles:                
+                if perm not in NO_ACQUIRE_PERMS:
+                    zport.manage_permission(perm, roles + [CZ_ADMIN_ROLE], 1)
+                else:
+                    zport.manage_permission(perm, roles + [CZ_ADMIN_ROLE], 0)
+
```

Comment:
Modifying existing migration scripts is not typically recommended, especially if it's already been released.  Any fixes should be applied in a later migration script.